### DISCUSSION TITLE
first install error theme

### DISCUSSION THREAD

**Discussion by vvchumanov** [prasanthrangan/hyprdots#1981] [Q&A]
After running the script for the first installation, the steps go correctly. But when it comes to installing patching themes it shows an error on Nordic blue --scipcaching false

**Comment by kRHYME7** [comment]
found the issue. Will update the script in a bit

@keth-real Can you please bump themepatcher.sh and install.sh. Thankss

**Comment by TowhidMolla-Dev** [comment]


**Reply by vvchumanov** [reply]
@TowhidMolla-Dev
Did you change your rights? 
`sudo chmod +X ./install_pkg.sh`

**Reply by TowhidMolla-Dev** [reply]
Thanks...But now I am also facing same issue as you

**Reply by kRHYME7** [reply]
git pull again maybe your local is outdated. then rerun install.sh -rs
